src/config/dynamic-theme-fixes.config

```diff
@@ -2898,32 +2898,83 @@ span[class^="burger"]
 codeforces.com
 
 INVERT
-#header > div:first-child img
-.action-link > div > img:first-child
-.roundbox-lt
-.roundbox-rt
-.roundbox-lb
-.roundbox-rb
+#userActivityGraph rect[data-items=""]
 .delete-resource-link
-a.contestParticipantCountLinkMargin > img
-.recent-actions > ul > li img[src*="hourglass"]
+.action-link img
+.non-decorated img[src$="tablesorter-bg.gif"]
 
 CSS
+#header > div:first-child > a::before {
+    content: "";
+    display: inline-block;
+    height: 65px;
+    width: 15px;
+    border-top-left-radius: 12px;
+    border-bottom-left-radius: 12px;
+    background-color: #fff;
+}
+#header > div:first-child > a img {
+    border-radius: 0 12px 12px 0;
+}
+#header > div:first-child img {
+    border-radius: 12px;
+}
+.datatable,
+.datatable > div,
+.roundbox {
+    border-radius: 6px;
+}
 .lt,
 .rt,
 .lb,
 .rb,
 .ilt,
-.irt {
+.irt,
+.roundbox-lt,
+.roundbox-rt,
+.roundbox-lb,
+.roundbox-rb {
     display: none;
 }
-input[type="submit"], input[type="button"], select {
-    border-style: solid;
+.backLava,
+.leftLava {
+    border-radius: 4.5px;
+}
+.backLava,
+.leftLava,
+.bottomLava,
+.cornerLava {
+    background: ${#BEBEBE} !important;
+}
```

Comment:
It just improves the performance of the default lava block. In the original version, this is a raster image, I replaced it with a vector-like HTML object. (Highlighted "GROUPS" in the screenshot)
![изображение](https://user-images.githubusercontent.com/38539949/145908185-abb9a6da-09d9-4eef-951b-69b96468126a.png)

Reply:
And also corrects its color, of course

Reply:
Default light version, unfixed dark version, fixed dark version:
![изображение](https://user-images.githubusercontent.com/38539949/146065171-d14de1c3-5a3c-4bf3-a738-cd1b640740cf.png)
![изображение](https://user-images.githubusercontent.com/38539949/146065274-47545d8f-86c6-466d-ae0a-d3706ba7eb98.png)
![изображение](https://user-images.githubusercontent.com/38539949/146065300-4c814876-1e5a-42b5-bac1-ab764c5b4cf8.png)

Reply:
It is a just a fix, as you can see

Reply:
So, I have fixed the color of this lava block. However, since I can not change the color of the image it was made of in the original version, I replaced it with an HTML object with color = `${original image color}`

Reply:
Oh woops, indeed a fix. Then it's fine 😅